### PR TITLE
Enable Laravel auto-discovery

### PR DESCRIPTION
**Pull request by sergiz**
Auto-discovery has now been available in Laravel for more than 2 years, so it would be nice, if this package also supported it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,8 @@ And then run `composer install` from the terminal.
 
 ## Configuration
 
+> If you are using Laravel 5.5+, you can skip these two steps. ([service providers and aliases are automatically registered](https://laravel.com/docs/6.0/packages#package-discovery)).
```
Comment (crynobone) [inline README.md]: This version target 7.0. If you install it with any lower version it still doesn't enable auto discovery.

File: composer.json [hunk 2]
```diff
@@ -34,6 +34,12 @@
     "extra": {
         "branch-alias": {
             "dev-master": "5.0-dev"
+        },
+        "providers": [
+            "Orchestra\\Asset\\AssetServiceProvider"
+        ],
+        "aliases": {
+            "Asset": "Orchestra\\Support\\Facades\\Asset"
```
Comment (crynobone) [inline composer.json]: I will not accept adding `aliases`.

**Comment by sergiz** [comment]
Meh, laravie also doesn't use auto-discovery for some reason (even though it's supposed to be a package made specifically for Laravel).

**Comment by crynobone** [comment]
> Meh, laravie also doesn't use auto-discovery for some reason (even though it's supposed to be a package made specifically for Laravel).

It doesn't because I need to have it be overriden by `orchestra/html` when installed under Orchestra Platform.

**Comment by sergiz** [comment]
Would that be a use case for `dont-discover` ?

**Comment by crynobone** [comment]
Only project composer.json can configure dont-discover, packages doesn't have control over other packages

**Pull request closed without merging**